compiler/rustc_codegen_ssa/src/back/symbol_export.rs

```diff
@@ -316,7 +303,7 @@ fn upstream_monomorphizations_provider(
                     // If there are multiple monomorphizations available,
                     // we select one deterministically.
                     let other_cnum = *e.get();
-                    if cnum_stable_ids[other_cnum] > cnum_stable_ids[cnum] {
+                    if tcx.stable_crate_id(other_cnum) > tcx.stable_crate_id(cnum) {
```

Comment:
As I understand this is a different order than used previously, but it's still deterministic so it's ok.